Replicator/tests/ReplicatorCollectionTest.cc

```diff
@@ -511,6 +511,96 @@ TEST_CASE_METHOD(ReplicatorCollectionTest, "Multiple Collections Incremental Pus
     validateCollectionCheckpoints(db, db2, 1, "{\"local\":22,\"remote\":24}");
 }
 
+struct Jthread {
```

Comment:
I have just noticed that there is `static std::thread* runInParallel(std::function<void()> callback)` in ReplicatorLoopbackTest.hh. Can `runInParallel` is used instead of Jthread? BTW, what is `J` stand for?

Reply:
c++20 introduced a new class, std::jthread. It is automatically joined when deleted. I just mimic it before we move to c++20.

Reply:
I'm curious why don't use runInParallel() instead of writing a new JThread? For this test, the test will not finish until the code running in the separate thread is done anyway. I don't see the benefit of using JThread.